tox.ini

```
@@ -4,9 +4,8 @@ usedevelop = True
 skip_missing_interpreters = True
 
 envlist = 
-    python{3.9,3.10,3.11}-django4.2-wagtail{5.2,6.3,6.4}
-    python{3.10,3.11,3.12}-django5.0-wagtail{5.2,6.3,6.4}
-    python{3.12,3.13}-django5.1-wagtail{6.3,6.4}
+    python{3.9,3.10,3.11,3.12}-django4.2-wagtail{5.2,6.3,6.4,7.0}
```

```suggestion
    python{3.9,3.10,3.11,3.12}-django4.2-wagtail{6.3,6.4,7.0}
```

**Suggestion**: Remove Wagtail 5 as EOL.